src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -568,8 +596,25 @@ def load_arguments(self, _):
                        '--config-file'], help='The config json file.', required=False)
             c.argument('weekday', options_list=[
                        '--weekday'], help='weekday on which maintenance can happen. e.g. Monday', required=False)
-            c.argument('start_hour', type=int, options_list=[
-                       '--start-hour'], help='maintenance start hour of 1 hour window on the weekday. e.g. 1 means 1:00am - 2:00am', required=False)
+            c.argument('start_hour', type=int, required=False, options_list=[
+                       '--start-hour'], help='maintenance start hour of 1 hour window on the weekday. e.g. 1 means 1:00am - 2:00am')
+            c.argument('schedule_type', arg_type=get_enum_type(schedule_types),
+                       help='Schedule type for non-default maintenance configuration.', required=False)
+            c.argument('interval_days', type=int, help='The number of days between each set of occurrences for Daily schedule.', required=False)
+            c.argument('interval_weeks', type=int, help='The number of weeks between each set of occurrences for Weekly schedule.', required=False)
+            c.argument('interval_months', type=int, help='The number of months between each set of occurrences for AbsoluteMonthly or RelativeMonthly schedule.', required=False)
+            c.argument('day_of_week', help='Specifies on which day of the week the maintenance occurs for Weekly or RelativeMonthly schedule.', required=False)
+            c.argument('day_of_month', help='Specifies on which date of the month the maintenance occurs for AbsoluteMonthly schedule.', required=False)
+            c.argument('week_index', arg_type=get_enum_type(week_indexes), required=False,
+                       help='Specifies on which instance of the weekday specified in --day-of-week the maintenance occurs for RelativeMonthly schedule.')
+            c.argument('duration_hours', options_list=['--duration'], type=int, required=False,
+                       help='The length of maintenance window. The value ranges from 4 to 24 hours.')
+            c.argument('utc_offset', validator=validate_utc_offset, required=False,
+                       help='The UTC offset in format +/-HH:mm. e.g. -08:00 or +05:30.')
+            c.argument('start_date', validator=validate_start_date, required=False,
+                       help='The date the maintenance window activates. e.g. 2023-01-01.')
+            c.argument('start_time', validator=validate_start_time, required=False,
+                       help='The start time of the maintenance window. e.g. 09:30.')
```

Comment:
> required=False

Same as above

Reply:
fixed~

Reply:
Update: I have to add `required=False` back in order to pass Linter and Live tests T^T . I posted the question in the Azure CLI channel: https://teams.microsoft.com/l/message/19:31160a5ec7e24ba0b9d58a5e812c43dd@thread.tacv2/1673502442579?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=9a661795-1b01-4a5b-9dd5-be571422334c&parentMessageId=1673502442579&teamName=Azure%20CLIs%20partners&channelName=Azure%20CLI&createdTime=1673502442579&allowXTenantAccess=false

Reply:
Update again: `required=False` are removed!